docs/README.md

```diff
@@ -13,6 +13,20 @@ If you don't have `pnpm` installed, you will need to do so first.
 pnpm i
 ```
 
+## Optional Packages
+
+If you would like to include consent management functionality in your local development environment,
+you can install the `@wcp/wcp-consent` package. This package is used for handling user consent
```

Comment:
I would recommend linking to documentation for this library, and then note that it's MS-internal.

Reply:
Yeah, I think I'd write all this from the perspective of "This bit is related to the production deployment of the site; if you're doing local development it's probably not relevant to you. If you're a Microsoft engineer who needs to work on this, refer to our internal documentation."

On related topics, using optional-dependencies does seem to work. If our pipelines set up the appropriate npm registries and credentials, the package will get installed when running there, and it'll just be ignored when installing the deps locally.

Reply:
General OSS guidance includes avoiding links or references to internal tooling/infra/documentation. I'd recommend you take what you wrote here and put it on a page on our internal documentation on the website (near discussion of its infrastructure), and just include prose that mentions MS employees can find more details there.